.github/workflows/fossa_scan.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: fossas/fossa-action@main
+      - uses: fossas/fossa-action@java21
```

Comment:
We should only change template.yml@main to template.yml@java21

Reply:
Fixed with 83c65f98306e0da9824443382f65f035661ea530